coreos-ostree-importer/coreos_ostree_importer.py

```diff
@@ -48,14 +49,34 @@
     "compose": "/mnt/koji/compose/ostree/repo",
 }
 
-
 class Consumer(object):
     def __init__(self):
-        # Check the possible repos to make sure they exist
-        for path in KNOWN_OSTREE_REPOS.values():
+        # Do sanity checks on the repos
+        for repo, path in KNOWN_OSTREE_REPOS.items():
+
+            # Check the repo to make sure it exists
             if not ostree_repo_exists(path):
                 raise Exception(f"OSTree repo does not exist at {path}")
 
+            # Sanity check the repo to make sure all directories in the repo
+            # are group writable. https://pagure.io/releng/issue/8811#comment-616490
+            # Wrap in try/except so we can hopefull see the logs before openshift
+            # deletes the pod
```

Comment:
Hmm, but OpenShift doesn't delete the pod on failure. It just restarts it. In this case, since we're re-raising the issue, it'll just restart it a bunch of time before failing, but the pod should stick around for us to fetch the logs. So I think we don't need to wrap this in a `try-except` block at all, right?

Reply:
when it starts a new pod it deletes the old one, doesn't it? I don't know, i've just had a ton if issues in the past when trying to get logs and the pods keep disappearing

Reply:
The default behaviour is to restart the same pod (i.e. not delete and create a new one). If it keeps restarting, it'll eventually stop with `CrashLoopBackOff` and just sit there.

Reply:
ok I'll remove the try/except